spec/unit/provider/service/launchd_spec.rb

```diff
@@ -3,6 +3,10 @@
 
 require 'spec_helper'
 
+if Puppet.features.microsoft_windows?
```

Comment:
Looks like the top of `launchd.rb` has this:

```
require 'puppet/util/plist' if Puppet.features.cfpropertylist?
```

That can likely be edited as you've done elsewhere....